Adapter/GADMAdapterMoPub.m

```diff
@@ -1,263 +0,0 @@
-  self.bannerAd.adUnitId = self.connector.publisherId;
-
-  [self.bannerAd loadAd];
-  NSLog(@"Requesting banner from Sample Ad Network");
-}
-
-
-- (BOOL)isBannerAnimationOK:(GADMBannerAnimationType)animType {
-    return YES;
-}
-
-// Loading Native Ads
-
-- (void)getNativeAdWithAdTypes:(NSArray *)adTypes options:(NSArray *)options {
-  
-  MPStaticNativeAdRendererSettings *settings = [MPStaticNativeAdRendererSettings alloc];
-  
-  MPNativeAdRendererConfiguration *config = [MPStaticNativeAdRenderer rendererConfigurationWithRendererSettings:settings];
-  NSString *moPubPublisherId = [self.connector credentials][@"ad_unit"]; // for facebook testing @"1ceee46ba9744155aed48ee6277ecbd6"; //
-    
-  MPNativeAdRequest *adRequest = [MPNativeAdRequest requestWithAdUnitIdentifier: moPubPublisherId rendererConfigurations:@[config]];
-                                                
-  MPNativeAdRequestTargeting *targeting = [MPNativeAdRequestTargeting targeting];
-    CLLocation *currentlocation = [[CLLocation alloc] initWithLatitude:self.connector.userLatitude longitude:self.connector.userLongitude];
-    targeting.location = currentlocation;
-    
-  [adRequest startWithCompletionHandler:^(MPNativeAdRequest *request, MPNativeAd *response, NSError *error) {
-      if (error) {
-          [self.connector adapter:self didFailAd:error];
-      }
-      else {
-          self.nativeAd = response;
-          self.nativeAd.delegate = self;
-      
-          if(options!=nil){
-    
-              for (GADNativeAdImageAdLoaderOptions *imageOptions in options) {
-                  if (![imageOptions isKindOfClass:[GADNativeAdImageAdLoaderOptions class]]) {
-                      continue;
-                  }
-                  
-                  // Load only image urls
-                  if(imageOptions.disableImageLoading)
-                  {
-                      _mediatedAd = [[MoPubAdapterMediatedNativeAd alloc] initWithMoPubNativeAd:self.nativeAd mappedImages:nil];
-                      [self.connector adapter:self didReceiveMediatedNativeAd:_mediatedAd];
-                  }
-              }
-          }
-        
-          [self loadNativeAdImages];
-
-      }
-
-  }];
-    
-  
-  NSLog(@"Requesting native ad from Sample Ad Network");
-}
-
-//Helper classes for downloading images
-
-- (void)loadNativeAdImages {
-    _imagesDictionary = [[NSMutableDictionary alloc] init];
-    // Load icon and cover image, and notify the  connector when completed.
-    NSURL *iconURL = [NSURL URLWithString:self.nativeAd.properties[kAdIconImageKey]];
-    NSURL *coverImageURL = [NSURL URLWithString:self.nativeAd.properties[kAdMainImageKey]];
-    [self loadCoverImageForURL:coverImageURL];
-    [self loadIconForURL:iconURL];
-}
-// Load main image for URL.
-- (void)loadCoverImageForURL:(NSURL *)coverImageURL {
-    __weak typeof(self) weakSelf = self;
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
```

Comment:
When using addDownloadImageURLs,how I can access the downloaded images? This method seems to be storing in the native cache automatically without giving access to the downloaded images. Can you please confirm.